lib/Dialect/MSFT/Transforms/PassCommon.cpp

```diff
@@ -54,7 +54,7 @@ StringRef circt::msft::getValueName(Value v, const SymbolCache &syms,
   if (auto blockArg = v.dyn_cast<BlockArgument>()) {
     auto portInfo =
         getModulePortInfo(blockArg.getOwner()->getParent()->getParentOp());
-    return portInfo.inputs[blockArg.getArgNumber()].getName();
+    return (portInfo.begin_input() + blockArg.getArgNumber())->getName();
```

Comment:
Yeah, this is another case where `at_input` is the more efficient way...

Reply:
It didn't exist yet when I wrote that.  But it isn't actually anymore efficient, just more concise.

Reply:
ahh... yeah. I was thinking that if ports allowed mixed order inputs/outputs the input iterator would have to iterate through from the beginning. So this'll be problematic perf in the future. Not relevant for now.